src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/List.cshtml

```diff
@@ -94,9 +105,12 @@
                 </div>
             </div>
         </div>
-        <div class="form-group col-6 float-right">
+        <div class="form-group col-5 float-right">
             <div class="dropdown float-right">
-                <button disabled class="btn btn-secondary dropdown-toggle" type="button" id="bulk-action-menu-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <button disabled class="btn btn-secondary dropdown-toggle d-none d-sm-block" type="button" id="bulk-action-menu-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    @T["Bulk Actions"]
+                </button>
+                <button disabled class="btn btn-sm btn-secondary dropdown-toggle d-block d-sm-none" type="button" id="bulk-action-menu-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                     @T["Bulk Actions"]
```

Comment:
This has two `id="bulk-action-menu-button"` in it now, so will break as @matiasmolleja explain on your other PR.

I agree with @matiasmolleja that making buttons smaller on mobile just makes them harder to click, especially for my fat fingers.

So not a big fan of the concept in general.

The reason bootstrap does not have responsive button classes is because they are not a fan of the concept either, it just makes for a bad user experience when the buttons are so small, you can't click them.

D-block buttons can be a good option for mobile, and flex-width them

If you are insisting on it though, there are options, you have to do it in sass though with the bootstrap mixins, otherwise you always have the two id's. The bs d-classes you are using, are just helpers wrapping display:none. 
So the dom still contains these two elements with duplicate id's, just one will be hidden visually.